examples/sat/sat_tensornetwork.py

```diff
@@ -79,7 +68,6 @@ def sat_tn(clauses: List[Tuple[int, int, int]]
   for clause in clauses:
     if 0 in clause:
       raise ValueError("0's are not allowed in the clauses.")
-  net = tensornetwork.TensorNetwork("tensorflow")
   var_set = set()
```

Comment:
Mypy failure: examples/sat/sat_tensornetwork.py:71: error: Need type annotation for 'var_set' (hint: "var_set: Set[<type>] = ...")

Reply:
My not going to try to support both mypy and pytype failures.